src/Particular.CodeRules/Particular.CodeRules.csproj

```diff
@@ -47,10 +131,13 @@
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
     <Error Condition="!Exists('..\packages\NuGetPackager.0.5.5\build\NuGetPackager.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NuGetPackager.0.5.5\build\NuGetPackager.targets'))" />
-    <Error Condition="!Exists('..\packages\GitVersionTask.1.3.2\build\GitVersionTask.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\GitVersionTask.1.3.2\build\GitVersionTask.targets'))" />
+    <Error Condition="!Exists('..\packages\GitVersionTask.3.3.0\build\dotnet\GitVersionTask.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\GitVersionTask.3.3.0\build\dotnet\GitVersionTask.targets'))" />
   </Target>
   <Import Project="..\packages\NuGetPackager.0.5.5\build\NuGetPackager.targets" Condition="Exists('..\packages\NuGetPackager.0.5.5\build\NuGetPackager.targets')" />
-  <Import Project="..\packages\GitVersionTask.1.3.2\build\GitVersionTask.targets" Condition="Exists('..\packages\GitVersionTask.1.3.2\build\GitVersionTask.targets')" />
+  <Import Project="..\packages\GitVersionTask.3.3.0\build\dotnet\GitVersionTask.targets" Condition="Exists('..\packages\GitVersionTask.3.3.0\build\dotnet\GitVersionTask.targets')" />
+  <PropertyGroup>
+    <PostBuildEvent>"$(SolutionDir)packages\NuGet.CommandLine.3.3.0\tools\NuGet.exe" pack $(SolutionDir)..\packaging\nuget\Particular.CodeRules.nuspec -OutputDirectory . -version 1.0.4-beta -prop configuration=$(ConfigurationName)</PostBuildEvent>
```

Comment:
The order of import matters. But if you got it right you should be able to access  `$(GitVersion_*)` where `* ` stands for all the information that GitVersion offers like `InformationalVersion`